### ISSUE TITLE
Shaders cause visual isssue's - A note about it

### ISSUE THREAD

**Issue by modmuss50** [modmuss50/OptiFabric#21]
# A message to Optifine users and developers

#### TL;DR: **OptiFabric is no longer supported.**

Please do not send messages to me, or the Optifine developers demanding fabric support.

## The Issue
A little over a week ago I released OptiFabric and it quickly came to my attention that there were some incompatibilities, especially with the shaders mod. I had a large amount of DM/pings letting me know of this.

The issue was tracked down to be an incompatibly with Fabric API's new Rendering API and the implementation called Indigo. To fix the issue correctly knowledge of how Optifine works is required, this is not possible without good knowledge of the fabric rendering API and details on how Optifine works.

I see a few solutions to this issue,

•	Official built in support for Fabric in Optifine. I think this is the best solution to the issue and the Optifine Developers should explore the idea further. (More on this later)

•	A fix from a developer outside of Fabric or Optifine could create a PR to OptiFabric, I would not mind looking into giving them access over the project if required.

•	Don’t use the features that don’t work, not great but it seems to work ok as-is currently. 

## The best solution

The ideal solution to this whole issue is official support for Fabric from Optifine. OptiFabric has proved that is possible and that users want the feature (5000+ downloads in week!). 

I see a few ways this could be made possible:

•	Optifine could include parts or the whole of OptiFabric, I see this being the easiest solution as the code already exists and has been proven to mostly work. 

•	Optifine could become a normal fabric mod that uses mixins. This would require a lot of time and effort from the Optifine Developers but it would allow faster updates to newer Minecraft versions.

If you want to explore this option or just need to ask a few questions feel free to contact me.

## The current Situation
As for now I can no longer maintain OptiFabric to the standard that I had intended to, and there will be no updates for the foreseeable future (Unless someone else can take on the challenge). Do not expect any support from Optifine or the Fabric developers if you continue to use this mod.

Thanks for reading this and understanding,

Modmuss50
